### PR TITLE
Enhanced the navbar in books.html

### PR DESCRIPTION
**Pull request by vansh-sardana**
# Related Issue
Fixes:  #1512

# Description
Corrected the navbar in books.html page. All the items in navbar are now properly aligned for all the screen sizes.
#1512 

# Type of PR

- [ ] Bug fix
- [x] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

# Screenshots / videos (if applicable)
Before fixing: 

https://github.com/anuragverma108/SwapReads/assets/107904311/0a5c5f80-6c4f-4ca7-848e-47f827e349a9

After fixing: 

https://github.com/anuragverma108/SwapReads/assets/107904311/41eb8a08-9d78-4ae7-8da1-10cb61abfa72

https://github.com/anuragverma108/SwapReads/assets/107904311/31803101-0708-4e5f-84a9-c9251b139559

# Checklist:
- [x] I have made this change from my own.
- [ ] I have taken help from some online resources.
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers and screenshots after making the changes.

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -3467,8 +3467,3 @@ body.dark-mode .h3 {
   .fab.fa-twitter{
     font-family:sans-serif;
   }
-  .fab.fa-twitter::before{
-    content:"𝕏";
```
Comment (vansh-sardana) [inline assets/css/style.css]: This part was deleted by mistake as a part of learning. in next commit, i have added this back.

**Comment by vansh-sardana** [comment]
What?

**Comment by vansh-sardana** [comment]
@RitiChandak Why have you removed that?

**Comment by vansh-sardana** [comment]
Does this really make sense removing labels after 3 weeks of closing? @RitiChandak

**Comment by RitiChandak** [comment]
Your changes were reverted

**Pull request merged**